www/doc/1.6/report.html

```diff
@@ -84,6 +84,18 @@ <h2 id="graph">Error graph</h2>
     input and output program (both are toggled on).</figcaption>
   </figure>
 
+  <h2 id="bogosity">Bogosity</h2>
+  <p>
+  Bogosity is a meter to glance at the validity of a program. Points 
+  sampled by Herbie will fall into one of the fallowing categories.
+  <ul>
+     <li>Valid - Correctly rounded</li>
+     <li>Invalid - Proven to be invalid sqrt(-1)</li>
+     <li>Infinite - Proven to have very large output</li>
+     <li>Unsamplable - Interval Analysis didn't converge</li>
+  </ul>
+  </p>
+
```

Comment:
Don't add it to the v1.6 documentation, because this won't appear in Herbie v1.6—it'll be released with v1.7.